Docs/Configuration.tex

```diff
@@ -1181,25 +1179,25 @@ \section{Booter}\label{booter}
 
 \subsection{Introduction}\label{booterintro}
 
-This section allows to apply different kinds of UEFI modifications on
-Apple bootloader (\texttt{boot.efi}). The modifications currently provide
-various patches and environment alterations for different firmware. Some
+This section allows the application of different types of UEFI modifications to
+the Apple bootloader (\texttt{boot.efi}). The modifications currently provide
```

Comment:
It no longer contains just Apple. Use "to the operating system bootloaders, mainly Apple bootloader (…`".